index.html

```diff
@@ -3651,39 +3656,81 @@ <h3>Parsing content</h3>
         </ol>
       </li>
       <li>
-        If |ndef|'s |typeNameField| is `1` ([=well-known type record=]):
-        <ol>
-          <li>
-            Set |record| to the result of the algorithm below switching on
-            |ndef|'s |type:string|:
+        If |ndef|'s |typeNameField| is `1` ([=well-known type record=]), then
+        switching on |ndef|'s |type:string|:
             <dl>
               <dt>"`T`" (`0x54`)</dt>
-              <ul>
+              <ol>
                 <li>
-                  Running <a>parse an NDEF text record</a> on |ndef|.
+                  Set |record| to the result of running
+                  <a>parse an NDEF text record</a> on |ndef|.
                 </li>
-              </ul>
+              </ol>
               <dt>"`U`" (`0x55`)</dt>
-              <ul>
+              <ol>
                 <li>
-                  Running <a>parse an NDEF URL record</a> on |ndef|.
+                  Set |record| to the result of running
+                  <a>parse an NDEF URL record</a> on |ndef|.
                 </li>
-              </ul>
+              </ol>
               <dt>"`Sp`" (`0x53` `0x70`)</dt>
-              <ul>
+              <ol>
                 <li>
-                  Running <a>parse an NDEF smart-poster record</a> on |ndef|.
+                  Set |record| to the result of running
+                  <a>parse an NDEF smart-poster record</a> on |ndef|.
                 </li>
-              </ul>
+              </ol>
+              <dt>"`:s`" (`0x53` `0x70`)</dt>
+              <ol>
+                <li>
+                  If |context| is not equal to `"smart-poster"`,
+                  [= exception/throw =] a {{TypeError}} and abort these steps.
+                </li>
+                <li>
+                  Set |record| to the result of running
+                  <a>parse a smart-poster size record</a> on |ndef|.
+                </li>
+              </ol>
+              <dt>"`:t`" (`0x53` `0x70`)</dt>
+              <ol>
+                <li>
+                  If |context| is not equal to `"smart-poster"`,
+                  [= exception/throw =] a {{TypeError}} and abort these steps.
+                </li>
+                <li>
+                  Set |record| to the result of running
+                  <a>parse a smart-poster type record</a> on |ndef|.
+                </li>
+              </ol>
+              <dt>"`:act`" (`0x53` `0x70`)</dt>
+              <ol>
+                <li>
+                  If |context| is not equal to `"smart-poster"`,
+                  [= exception/throw =] a {{TypeError}} and abort these steps.
+                </li>
+                <li>
+                  Set |record| to the result of running
+                  <a>parse a smart-poster action record</a> on |ndef|.
+                </li>
+              </ol>
```

Comment:
How about other local type records that may contained in a smart-poster record.
Nothing prevents that if I understand it correctly.

Reply:
Fixed, thanks. I had to rewrite it quite much :).